Dockerfile

```diff
@@ -0,0 +1,35 @@
+
+
+
+FROM ubuntu:yakkety
```

Comment:
Could you switch to 14.04/trusty?  I know it is old but that is what I use for production builds since it seems to provide bins with the broadest compatability.